src/stylesheets/blocks/_header.scss

```diff
@@ -1 +1,143 @@
+.header {
+  background-color: color(mint-green);
+  width: 100%;
+}
+
+.header-wrapper {
+  display: block;
+  margin: 0 auto;
+  max-width: rem(680);
+  position: relative;
+}
+
+.header__background-img {
+  background-repeat: no-repeat; 
+  background-size: cover;
+  display: inline-block;
+  position: absolute;
+}
+
+.img-1 {
```

Comment:
i'd prefer to see these nested under `.header__background-img` as `&.-img-1` or something. for a small site this may be okay but it's not scalable -- `.img-1` is a vague name ripe for collisions